code/modules/projectiles/updated_projectiles/guns/specialist.dm

```diff
@@ -357,12 +361,20 @@
 	damage_mult = config.base_hit_damage_mult
 
 
+/obj/item/weapon/gun/launcher/m92/wield(mob/living/user)
+	. = ..()
+	if(user.mind?.cm_skills && user.mind.cm_skills.spec_weapons < 0)
+		do_after(user, 10, TRUE, 5, BUSY_ICON_HOSTILE)
```

Comment:
Are you sure this works as intended? This seems to have no effect. It should be as it is in the after-attack.